examples/android/android-studio/Cukeulator/README.md

```diff
@@ -0,0 +1,72 @@
+## Cukeulator Example Test
+This is the example test-project for the Cukeulator app for Android Studio (Beta) 0.8.14.
+
+### Setup
+Features must be placed in `assets/features/`. Subdirectories are allowed.
+
+The rest of the dependencies are added automatically in `app/build.gradle`.
+
+The cucumber-android dependency is added as:
+
+```
+androidTestCompile 'info.cukes:cucumber-android:1.2.0@jar'
```

Comment:
@mdzyuba this is due to the fact that there is an apklib and a jar file on maven central and jcenter.

@brasmusson / @aslakhellesoy  do you know why the project was specified to be packaged as apklib? I think plain jar is sufficient.